src/api/action/profile/get.ts

```diff
@@ -1,6 +1,7 @@
 import { isActor } from '@fedify/fedify';
 import type { Context } from 'hono';
 
+import { lookupObject } from 'lookup-helpers';
```

Comment:
Should we use relative path to resolve this? I hit a weird issue the other week where we ended up utilising a package and helper with the same name and the wrong one was being imported and it wasn't totally clear what the issue was 😅

Reply:
Yeah good shout - my typescript lang server keeps using absolute tho!